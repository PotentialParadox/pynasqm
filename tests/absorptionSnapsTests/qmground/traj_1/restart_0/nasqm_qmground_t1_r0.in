300K constant temp QMMMMD
 &cntrl
  imin=0,
  iwrap=1,
  irest=0,
  ntx=5,
  ntb=1, ! 1-Constant Volume, 2-Pressure
  ntp=0,
  ig=-1,
  taup=2.0,
  cut=8.0,
  ntr=0,
  tempi=300.0,
  temp0=300.0,
  ntt=0, ! 0-Constant-Energy, 1-Weak-Coupling, 2-Anderson, 3-Langevin
  gamma_ln=0, ! must be 0 if ntt < 3
  nstlim=10,
  dt=0.0005,
  ntpr=1,
  ntwx=2,
  ntwv=-1,
  ifqnt=1
 /
 &qmmm
  scfconv=1.0000E-10,
  verbosity=1,
  qmmask=':1',
  diag_routine=1,
  qmcharge=0,
  qm_theory='AM1',
  qmshake=0,
  printdipole=1,
  qm_ewald=0,
  qm_pme=0,
 /
