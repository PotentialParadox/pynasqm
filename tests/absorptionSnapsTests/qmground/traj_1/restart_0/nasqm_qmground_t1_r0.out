
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 02/12/2020 at 12:08:29

|   Executable path: sander
| Working directory: /home/dustin/Documents/TestRuns/qm_mm_prop/benzene_nh2/qmground/traj_1/restart_0
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: nasqm_qmground_t1_r0.in                                               
| MDOUT: nasqm_qmground_t1_r0.out                                              
|INPCRD: snap_for_qmground_t1_r0.rst                                           
|  PARM: m1.prmtop                                                             
|RESTRT: snap_for_qmground_t1_r1.rst                                           
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: nasqm_qmground_t1_r0.nc                                               
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

300K constant temp QMMMMD                                                      
 &cntrl                                                                        
  imin=0,                                                                      
  iwrap=1,                                                                     
  irest=0,                                                                     
  ntx=5,                                                                       
  ntb=1, ! 1-Constant Volume, 2-Pressure                                       
  ntp=0,                                                                       
  ig=-1,                                                                       
  taup=2.0,                                                                    
  cut=8.0,                                                                     
  ntr=0,                                                                       
  tempi=300.0,                                                                 
  temp0=300.0,                                                                 
  ntt=0, ! 0-Constant-Energy, 1-Weak-Coupling, 2-Anderson, 3-Langevin          
  gamma_ln=0, ! must be 0 if ntt < 3                                           
  nstlim=10,                                                                   
  dt=0.0005,                                                                   
  ntpr=1,                                                                      
  ntwx=2,                                                                      
  ntwv=-1,                                                                     
  ifqnt=1                                                                      
 /                                                                             
 &qmmm                                                                         
  scfconv=1.0000E-10,                                                          
  verbosity=1,                                                                 
  qmmask=':1',                                                                 
  diag_routine=1,                                                              
  qmcharge=0,                                                                  
  qm_theory='AM1',                                                             
  qmshake=0,                                                                   
  printdipole=1,                                                               
  qm_ewald=0,                                                                  
  qm_pme=0,                                                                    
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
|  INFO: Old style inpcrd file read

| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    22.679
| New format PARM file being parsed.
| Version =    1.000 Date = 09/26/18 Time = 13:58:55
 NATOM  =    4394 NTYPES =       7 NBONH =     883 MBONA  =    2635
 NTHETH =    2641 MTHETA =    2636 NPHIH =      26 MPHIA  =       9
 NHPARM =       0 NPARM  =       0 NNB   =    9698 NRES   =     877
 NBONA  =    2635 NTHETA =    2636 NPHIA =       9 NUMBND =       6
 NUMANG =       7 NPTRA  =       3 NATYP =       7 NPHB   =       0
 IFBOX  =       1 NMXRS  =      14 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real              373094
|     Hollerith          14061
|     Integer           159675
|     Max Pairs        1464666
|     nblistReal         52728
|     nblist Int        368993
|       Total            11168 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
LOADING THE QUANTUM ATOMS AS GROUPS
     Mask :1; matches    14 atoms

     BOX TYPE: RECTILINEAR
Note: ig = -1. Setting random seed to   696304 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

m1                                                                              

General flags:
     imin    =       0, nmropt  =       0

Nature and format of input:
     ntx     =       5, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =       1, ntrx    =       1, ntwr    =      10
     iwrap   =       1, ntwx    =       2, ntwv    =      -1, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Molecular dynamics:
     nstlim  =        10, nscm    =      1000, nrespa  =         1
     t       =   0.00000, dt      =   0.00050, vlimit  =  20.00000

Initial temperature generation:
     ig      =  696304
     tempi   = 300.00000

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       1
     Box X =   49.256   Box Y =   50.133   Box Z =   45.358
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   50       NFFT2 =   50       NFFT3 =   45
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

QMMM options:
             ifqnt = True       nquant =       14
              qmgb =        0  qmcharge =        0   adjust_q =        2
              spin =        1     qmcut =   8.0000    qmshake =        0
          qmmm_int =        1
     lnk_atomic_no =        1   lnk_dis =   1.0900 lnk_method =        1
          qm_theory =     AM1 verbosity =        1
            qmqmdx = Analytical
      tight_p_conv = False (converge density to 0.05xSqrt[SCFCRT])
           scfconv = 0.100E-09  itrmax =     1000
      printcharges = False      printdipole = QM         peptide_corr = False
    qmmmrij_incore = True 
  qmqm_erep_incore = True 
       pseudo_diag = True pseudo_diag_criteria =   0.0500
      diag_routine =        1
          qm_ewald =        0 qm_pme = False 

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

Cpptraj Generated Restart                                                       
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0
 identifying link atoms
 identified link atoms

     Sum of charges from parm topology file =   0.00000100
     Forcing neutrality...
QMMM: ADJUSTING CHARGES
QMMM: ----------------------------------------------------------------------
QMMM: adjust_q = 2
QMMM: Uniformly adjusting the charge of MM atoms to conserve total charge.
QMMM:                             qm_charge =    0
QMMM: QM atom RESP charge sum (inc MM link) =    0.000
QMMM: Adjusting each MM atom resp charge by =    0.000
QMMM:          Sum of MM + QM region is now =    0.000
QMMM: ----------------------------------------------------------------------
|  # of SOLUTE  degrees of freedom (RNDFP):   13182.
|  # of SOLVENT degrees of freedom (RNDFS):       0.
|  QMSHAKE_NOSHAKEMASK_OVERLAP =      0
|  NDFMIN =   13179.     NUM_NOSHAKE =      0     CORRECTED RNDFP =   13179.
|  TOTAL # of degrees of freedom (RNDF) =   13179.
 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8332E-11   at   2.782960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =     349554
| TOTAL SIZE OF NONBOND LIST =     349554

QMMM: Link Atom Information
QMMM: ------------------------------------------------------------------------
QMMM: nlink =     0
QMMM: ------------------------------------------------------------------------

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       -0.3600   -0.4593   -1.3883
  QMMM:     2        2      C        0.7026   -1.0600   -0.7355
  QMMM:     3        3      C        1.1365   -0.6353    0.4887
  QMMM:     4        4      C        0.4805    0.4607    1.0763
  QMMM:     5        5      C       -0.6491    1.0810    0.4029
  QMMM:     6        6      C       -1.0642    0.5953   -0.8191
  QMMM:     7        7      H       -0.6253   -0.8669   -2.3660
  QMMM:     8        8      H        1.2057   -1.9295   -1.1571
  QMMM:     9        9      H        1.9372   -1.1532    1.0066
  QMMM:    10       10      H       -1.1440    1.8250    1.0098
  QMMM:    11       11      H       -1.9372    1.0132   -1.3195
  QMMM:    12       12      N        0.9482    1.0559    2.2184
  QMMM:    13       13      H        1.9284    1.2398    2.2688
  QMMM:    14       14      H        0.4784    1.9295    2.3660

--------------------------------------------------------------------------------
  3.1 QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 18

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is disabled.
| QMMM: Using internal diagonalization routine (diag_routine=1).

| QMMM: Estimated QM Dynamic Memory Usage (per thread)
| QMMM: ---------------------------------------------------
| QMMM:              QM Atom Type Info :          800 bytes
| QMMM:         QM RESP Charge Storage :          112 bytes
| QMMM:            QM Atom Number List :           56 bytes
| QMMM:                Link Atom Pairs :            0 bytes
| QMMM:          QM Atomic Number List :           56 bytes
| QMMM:                QM-MM Pair List :        17524 bytes
| QMMM:                   QM Atom Mask :         8788 bytes
| QMMM:           QM Coordinate Arrays :       140944 bytes
| QMMM:         Scaled MM Charge Array :        35152 bytes
| QMMM:    SCF Mulliken Charge Storage :          112 bytes
| QMMM:                QM Force Arrays :       105792 bytes
| QMMM:                 Density Matrix :         5040 bytes
| QMMM:          Density Matrix Copies :         5320 bytes
| QMMM: Fock2 Density Matrix Workspace :         1792 bytes
| QMMM:                    Fock Matrix :         5040 bytes
| QMMM:           Eigen Vector Storage :         9800 bytes
| QMMM: QM-QM Elec Repulsion Integrals :        16016 bytes
| QMMM:  QM 2-Elec Repulsion Integrals :        20888 bytes
| QMMM:              1-Electron Matrix :         5040 bytes
| QMMM:       _REAL_ parameter storage :        38584 bytes
| QMMM:      integer parameter storage :          448 bytes
| QMMM:         QM-MM RIJ Eqns storage :       207240 bytes
| QMMM:          _REAL_ Scratch arrays :       164472 bytes
| QMMM:         Integer Scratch arrays :        55176 bytes
| QMMM: ---------------------------------------------------
| QMMM:        Total Dynamic Memory Usage:      0.805 Mb

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

QMMM: SCF Converged to 0.1000E-09 in:    23 Cycles 
QMMM:
QMMM: SCF Energy =       20.43266241 KCal/mol,        85.49025954 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        0   TIME(PS) =       0.000  TEMP(K) =   300.39  PRESS =     0.0
 Etot   =     -1851.7678  EKtot   =      3933.4843  EPtot      =     -5785.2522
 BOND   =       925.1582  ANGLE   =      1421.4658  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5640.0187
 EELEC  =     -2512.2901  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        20.4327
 Ewald error estimate:   0.4455E-03
 ------------------------------------------------------------------------------

QMMM: SCF Converged to 0.1000E-09 in:     3 Cycles 
QMMM:
QMMM: SCF Energy =       20.43266241 KCal/mol,        85.49025954 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        1   TIME(PS) =       0.001  TEMP(K) =   300.39  PRESS =     0.0
 Etot   =     -1851.7678  EKtot   =      3933.4843  EPtot      =     -5785.2522
 BOND   =       925.1582  ANGLE   =      1421.4658  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5640.0187
 EELEC  =     -2512.2901  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        20.4327
 Ewald error estimate:   0.4455E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.846    1.259   -0.311    1.549
QMMM: SCF Converged to 0.1000E-09 in:    17 Cycles 
QMMM:
QMMM: SCF Energy =       20.67861418 KCal/mol,        86.51932174 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        2   TIME(PS) =       0.001  TEMP(K) =   302.51  PRESS =     0.0
 Etot   =     -1852.5533  EKtot   =      3961.2341  EPtot      =     -5813.7874
 BOND   =       887.6460  ANGLE   =      1430.3076  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5638.7900
 EELEC  =     -2513.6297  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        20.6786
 Ewald error estimate:   0.4855E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.853    1.240   -0.329    1.541
QMMM: SCF Converged to 0.1000E-09 in:    17 Cycles 
QMMM:
QMMM: SCF Energy =       20.96082232 KCal/mol,        87.70008060 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        3   TIME(PS) =       0.002  TEMP(K) =   302.89  PRESS =     0.0
 Etot   =     -1852.9594  EKtot   =      3966.2121  EPtot      =     -5819.1715
 BOND   =       872.8492  ANGLE   =      1439.0070  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5637.6302
 EELEC  =     -2514.3582  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        20.9608
 Ewald error estimate:   0.4820E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.859    1.220   -0.352    1.532
QMMM: SCF Converged to 0.1000E-09 in:    19 Cycles 
QMMM:
QMMM: SCF Energy =       21.30145630 KCal/mol,        89.12529314 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        4   TIME(PS) =       0.002  TEMP(K) =   301.16  PRESS =     0.0
 Etot   =     -1852.7859  EKtot   =      3943.5378  EPtot      =     -5796.3237
 BOND   =       886.1849  ANGLE   =      1447.1110  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5636.4923
 EELEC  =     -2514.4288  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        21.3015
 Ewald error estimate:   0.4643E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.862    1.199   -0.378    1.524
QMMM: SCF Converged to 0.1000E-09 in:    19 Cycles 
QMMM:
QMMM: SCF Energy =       21.85358309 KCal/mol,        91.43539166 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        5   TIME(PS) =       0.003  TEMP(K) =   297.59  PRESS =     0.0
 Etot   =     -1851.9390  EKtot   =      3896.7937  EPtot      =     -5748.7327
 BOND   =       924.4954  ANGLE   =      1454.2215  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5635.4611
 EELEC  =     -2513.8421  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        21.8536
 Ewald error estimate:   0.4671E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.890    1.107   -0.464    1.494
QMMM: SCF Converged to 0.1000E-09 in:    19 Cycles 
QMMM:
QMMM: SCF Energy =       22.29711494 KCal/mol,        93.29112890 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        6   TIME(PS) =       0.003  TEMP(K) =   293.03  PRESS =     0.0
 Etot   =     -1850.9772  EKtot   =      3837.0983  EPtot      =     -5688.0754
 BOND   =       976.9473  ANGLE   =      1460.0259  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5634.6896
 EELEC  =     -2512.6562  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        22.2971
 Ewald error estimate:   0.4653E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.888    1.088   -0.495    1.489
QMMM: SCF Converged to 0.1000E-09 in:    18 Cycles 
QMMM:
QMMM: SCF Energy =       22.74160122 KCal/mol,        95.15085952 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        7   TIME(PS) =       0.004  TEMP(K) =   288.66  PRESS =     0.0
 Etot   =     -1850.0112  EKtot   =      3779.9040  EPtot      =     -5629.9152
 BOND   =      1028.2425  ANGLE   =      1464.3194  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5634.2485
 EELEC  =     -2510.9701  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        22.7416
 Ewald error estimate:   0.4628E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.884    1.071   -0.528    1.486
QMMM: SCF Converged to 0.1000E-09 in:    18 Cycles 
QMMM:
QMMM: SCF Energy =       23.13225994 KCal/mol,        96.78537557 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        8   TIME(PS) =       0.004  TEMP(K) =   285.65  PRESS =     0.0
 Etot   =     -1849.2730  EKtot   =      3740.4686  EPtot      =     -5589.7416
 BOND   =      1063.1827  ANGLE   =      1467.0141  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5634.1489
 EELEC  =     -2508.9217  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        23.1323
 Ewald error estimate:   0.4664E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.877    1.057   -0.561    1.484
QMMM: SCF Converged to 0.1000E-09 in:    17 Cycles 
QMMM:
QMMM: SCF Energy =       23.41688960 KCal/mol,        97.97626608 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =        9   TIME(PS) =       0.005  TEMP(K) =   284.80  PRESS =     0.0
 Etot   =     -1849.0997  EKtot   =      3729.3401  EPtot      =     -5578.4398
 BOND   =      1071.2261  ANGLE   =      1468.1347  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5634.5407
 EELEC  =     -2506.6768  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        23.4169
 Ewald error estimate:   0.4915E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.869    1.047   -0.592    1.484
QMMM: SCF Converged to 0.1000E-09 in:    16 Cycles 
QMMM:
QMMM: SCF Energy =       23.56112562 KCal/mol,        98.57974957 KJ/mol
QMMM: SCF Energy = Heat of formation

 NSTEP =       10   TIME(PS) =       0.005  TEMP(K) =   286.32  PRESS =     0.0
 Etot   =     -1849.4467  EKtot   =      3749.2117  EPtot      =     -5598.6584
 BOND   =      1049.6736  ANGLE   =      1467.8012  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5635.2675
 EELEC  =     -2504.4269  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        23.5611
 Ewald error estimate:   0.4996E-03
 ------------------------------------------------------------------------------

                  X        Y        Z     TOTAL  
  QM DIPOLE (Debye)    0.860    1.039   -0.622    1.485

      A V E R A G E S   O V E R      10 S T E P S


 NSTEP =       10   TIME(PS) =       0.005  TEMP(K) =   294.30  PRESS =     0.0
 Etot   =     -1851.0813  EKtot   =      3853.7285  EPtot      =     -5704.8098
 BOND   =       968.5606  ANGLE   =      1451.9408  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =     -5636.1288
 EELEC  =     -2511.2201  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=        22.0376
 Ewald error estimate:   0.4730E-03
 ------------------------------------------------------------------------------


      R M S  F L U C T U A T I O N S


 NSTEP =       10   TIME(PS) =       0.005  TEMP(K) =     7.06  PRESS =     0.0
 Etot   =         1.4433  EKtot   =        92.4348  EPtot      =        93.8601
 BOND   =        74.8493  ANGLE   =        15.9754  DIHED      =         0.0000
 1-4 NB =         0.0000  1-4 EEL =         0.0000  VDWAALS    =         1.9451
 EELEC  =         3.2889  EHBOND  =         0.0000  RESTRAINT  =         0.0000
 AM1ESCF=         1.1019
|E(PBS) =         0.0647
 Ewald error estimate:   0.1535E-04
 ------------------------------------------------------------------------------


--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.01 ( 2.34% of Total)
|    Fast Water setup           0.00 ( 0.01% of Total)
|                Build the list             0.02 (67.28% of List )
|                Other                      0.01 (32.72% of List )
|             List time                  0.03 (13.00% of Nonbo)
|                   Short_ene time             0.09 (98.01% of Direc)
|                   Other                      0.00 ( 1.99% of Direc)
|                Direct Ewald time          0.09 (53.88% of Ewald)
|                Adjust Ewald time          0.00 ( 2.27% of Ewald)
|                Self Ewald time            0.00 ( 0.01% of Ewald)
|                   Fill Bspline coeffs        0.00 ( 2.95% of Recip)
|                   Fill charge grid           0.01 ( 8.84% of Recip)
|                   Scalar sum                 0.02 (22.16% of Recip)
|                   Grad sum                   0.01 (16.37% of Recip)
|                   FFT time                   0.04 (49.32% of Recip)
|                   Other                      0.00 ( 0.37% of Recip)
|                Recip Ewald time           0.07 (43.69% of Ewald)
|                Force Adjust               0.00 ( 0.03% of Ewald)
|                Virial junk                0.00 ( 0.11% of Ewald)
|                Other                      0.00 ( 0.01% of Ewald)
|             Ewald time                 0.17 (87.00% of Nonbo)
|          Nonbond force              0.19 (83.58% of Force)
|             QMMM setup                 0.00 ( 1.49% of QMMM )
|             QMMM list build            0.00 ( 1.57% of QMMM )
|             QMMM prep coords           0.00 ( 4.53% of QMMM )
|             QMMM RIJ Eqns Calc         0.00 ( 3.54% of QMMM )
|                   QMMM hcore QM-QM           0.00 (79.73% of QMMM )
|                   QMMM hcore QM-MM           0.00 (20.21% of QMMM )
|                   Other                      0.00 ( 0.06% of QMMM )
|                QMMM hcore calc            0.00 (13.42% of QMMM )
|                   QMMM fock build            0.01 (34.58% of QMMM )
|                   QMMM elec-energy cal       0.00 ( 0.70% of QMMM )
|                   QMMM full matrix dia       0.01 (25.78% of QMMM )
|                   QMMM pseudo matrix d       0.01 (32.78% of QMMM )
|                   QMMM density build         0.00 ( 5.59% of QMMM )
|                   Other                      0.00 ( 0.57% of QMMM )
|                QMMM scf                   0.02 (86.54% of QMMM )
|                Other                      0.00 ( 0.04% of QMMM )
|             QMMM energy                0.02 (71.90% of QMMM )
|             QMMM QM-QM force           0.00 ( 9.28% of QMMM )
|             QMMM QM-MM force           0.00 ( 6.86% of QMMM )
|             QMMM Collate Forces        0.00 ( 0.11% of QMMM )
|             Other                      0.00 ( 0.72% of QMMM )
|          QMMM                       0.03 (14.73% of Force)
|          Bond/Angle/Dihedral        0.00 ( 1.66% of Force)
|          Other                      0.00 ( 0.04% of Force)
|       Force time                 0.23 (97.60% of Runmd)
|       Verlet update time         0.01 ( 2.23% of Runmd)
|       Other                      0.00 ( 0.17% of Runmd)
|    Runmd Time                 0.24 (85.43% of Total)
|    Other                      0.03 (12.22% of Total)
| Total time                 0.28 (100.0% of ALL  )

| Number of list builds   :          1

| Highest rstack allocated:     366766
| Highest istack allocated:       4394

| Final Performance Info:
| -----------------------------------------------------
| Average timings for all steps:
|     Elapsed(s) =       0.24 Per Step(ms) =      23.75
|         ns/day =       1.82   seconds/ns =   47498.80
| -----------------------------------------------------

|           Job began  at 12:08:29.663  on 02/12/2020
|           Setup done at 12:08:29.704  on 02/12/2020
|           Run   done at 12:08:29.941  on 02/12/2020
|     wallclock() was called    2948 times
ASSERTion 'ier == 0' failed in src/sqm/qmmm_module.F90 at line    892.
